holidays/countries/singapore.py

```diff
@@ -96,7 +96,7 @@ def _populate(self, year):
         # Hari Raya Puasa
         # aka Eid al-Fitr
         # date of observance is announced yearly
-        dates_obs = {
+        dates_fixed_multiple_obs: Dict[int, List[Tuple[int, int]]] = {
```

Comment:
It seems that a tuple would be sufficient here (and below): Dict[int, Tuple[Tuple[int, int]]].

It also looks like the dict can be simplified to Dict[int, Tuple[int, int]] format.

Reply:
Yeah, so I thought at first glance, forgetting in the moment why I had coded it this way in the first place.

I quickly remembered that the Islamic calendar year is shorter than the solar Gregorian calendar year by 10 to 12 days so there is a chance that these Islamic holidays can fall twice in the same Gregorian year, which I stumbled upon when doing extensive code testing.  So it cannot be simplified.

I will add a remark so we won't forget in the future!

Thanks.

Reply:
Oh missed the part suggesting using a tuple instead of a list! Makes a ton of sense; will prepare a PR for it.